### PR TITLE
Address analysis issues from pub.dev

### PR DESCRIPTION
**Pull request by mikolak**

### CODE REVIEW

File: lib/blemulator.dart [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/// Library with all BLEmulator definitions
```
Comment (bartoszwilk) [inline lib/blemulator.dart:1]: Does the amount of slash make a difference? I noticed that in some places comment starts with double slash and in the others it starts with triple slash.

File: lib/blemulator_core.dart [hunk 2]
```diff
@@ -1,11 +1,13 @@
 part of blemulator;
 
+/// Entry point for using the simulator
 class Blemulator {
 
   static Blemulator _instance = Blemulator._internal();
 
   DartToPlatformBridge _toPlatformBridge;
   SimulationManager _simulationManager;
+  // ignore: unused_field
```
Comment (bartoszwilk) [inline lib/blemulator_core.dart:10]: Why can't we remove it then? It will not be hard in the future to add it again.

File: lib/blemulator_core.dart [hunk 3]
```diff
@@ -16,13 +18,14 @@ class Blemulator {
     _toDartBridge = PlatformToDartBridge(_simulationManager);
   }
 
+  /// Switches the implementation of the native adapter to simulation
```
Comment (bartoszwilk) [inline lib/blemulator_core.dart:21]: ...to the simulated one?

**Review by bartoszwilk — DISMISSED** [review]

File: lib/blemulator.dart (hunk 1 above)
Comment (mikolak) [reply lib/blemulator.dart:1]: `///` are C#-style dart docs

File: lib/blemulator_core.dart (hunk 2 above)
Comment (mikolak) [reply lib/blemulator_core.dart:10]: Discussed in person.
tl;dr: constructor of PlatformToDartBridge registers itself as a method call handler on a method channel and it would look weird to just call a constructor without any assignment of the result object to a field.

**Comment by pawelByszewski** [comment]
looks good but has some conflicts

**Review by pawelByszewski — APPROVED** [review]
